### PR TITLE
#166840921 Create user profile 

### PR DESCRIPTION
**Pull request by kechyy**
#### What does this PR do?
- Create user profile

#### Description of Task to be completed?
- update user profile route
- generate a test  random jwt token with email to authorise user
- create test to verify the accuracy of the return status code.

#### How should this be manually tested?
- clone the repo
- run npm install
- Hit npm start:dev to start server at port 3000
- Login
- Visit http://localhost:3000/api/v1/auth/user/profile

#### Any background context you want to provide?
- Must login with a valid email address
#### What are the relevant pivotal tracker stories?
[#166840921](https://www.pivotaltracker.com/story/show/166840921)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

File: .nyc_output/processinfo/index.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
+{
```
Comment (tolumide-ng) [inline .nyc_output/processinfo/index.json]: Great work @kechyy, could you please remove the .nyc_output folder 
```git rm -rf .nyc_output``` would work

File: .src/database/seeders/20190705223532-user.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+'use strict';
```
Comment (tolumide-ng) [inline .src/database/seeders/20190705223532-user.js]: Hello @kechyy , there seem to be four empty seeders here

File: src/test/user.spec.js [hunk 3]
```diff
@@ -192,3 +193,157 @@ describe('Test user login, signup and account verification', () => {
...
+        .set('x-access-token', validEmailToken)
+        .end((err, res) => {
+          expect(res.status).to.equal(200);
+          expect(res.body.message).to.equal('Record successfully updated');
+          done();
+        });
+    });
+  });
+
+  describe('GET /auth/view/profile/:userId', () => {
```
Comment (segunolalive) [inline src/test/user.spec.js]: Use Restful URL `/users/:id`, i.e `collection/collectionItem`

**Review by segunolalive — CHANGES_REQUESTED** [review]

File: src/controllers/user.controller.js [hunk 4]
```diff
@@ -170,6 +170,90 @@ class UserController {
...
+  /**
+   *
+   *
+   * @static
+   * @param {object} req - express request object
+   * @param {object} res - express response object
+   * @returns {object} - return user's updated data
+   * @memberof UserController
+   */
+  static async updateAccount(req, res) {
```
Comment (codexempire) [inline src/controllers/user.controller.js]: Please I think to drive the work of this `function` you could change the name to `updateAccountProfile` so anyone will know what the controller does to the accounts specifically

Comment (Onyimatics) [reply src/controllers/user.controller.js]: yea @kechyy, or even `updateProfile`

**Review by Onyimatics — CHANGES_REQUESTED** [review]
Good job @kechyy. I like the way you implemented your tests.
Please make the changes as suggested.

**Review by Onyimatics — APPROVED** [review]

**Review by timi-codes — APPROVED** [review]
Well done! @kechyy on the test.

**Pull request merged**